### PR TITLE
Parameterize Prometheus Exporter Docker image.

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

File: operator/templates/stateful-set.yaml [hunk 1]
```diff
@@ -150,8 +150,8 @@ spec:
               subPath: node-readiness-probe.sh
         {{ if eq .Params.PROMETHEUS_EXPORTER_ENABLED "true" }}
         - name: prometheus-exporter
-          imagePullPolicy: Always
-          image: mesosphere/cassandra-prometheus-exporter:0.1.0-2.2.1-SNAPSHOT
+          image: {{ PROMETHEUS_EXPORTER_DOCKER_IMAGE }}
```
Comment (viivek46) [inline operator/templates/stateful-set.yaml]:
```suggestion
          image: {{ .Params.PROMETHEUS_EXPORTER_DOCKER_IMAGE }}
```

File: operator/templates/stateful-set.yaml [hunk 2]
```diff
@@ -150,8 +150,8 @@ spec:
...
+          imagePullPolicy: {{ PROMETHEUS_EXPORTER_DOCKER_IMAGE_PULL_POLICY }}
```
Comment (viivek46) [inline operator/templates/stateful-set.yaml]:
```suggestion
          imagePullPolicy: {{ .Params.PROMETHEUS_EXPORTER_DOCKER_IMAGE_PULL_POLICY }}
```

**Comment by mpereira** [comment]
> why we are exposing Prometheus image in param.

We want to make it configurable at the operator level.

> also as we have made changes here should we revert back changes from '/templates' and metadata ?

I'll update the Docker image metadata handling on a separate PR. Maybe we'll make `params.yaml` a template.

**Comment by mpereira** [comment]
Thanks for fixing the parameter accessors!

**Pull request merged**